### PR TITLE
Update README with basic instructions.

### PR DESCRIPTION
**Pull request by thodges-gh**
Added instructions to build, test, and deploy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
 # solidity-cborutils
 Utilities for encoding [CBOR](http://cbor.io/) data in solidity
+
+## Install
+
+```bash
+$ git clone https://github.com/smartcontractkit/solidity-cborutils.git
+$ cd solidity-cborutils
+$ yarn install
```
Comment (Arachnid) [inline README.md]: npm install?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+      gas: 4700000
+    }
+  }
+};
+```
+
+Save, then run the following to deploy:
+
+```bash
+$ truffle migrate --reset --network devnet
```
Comment (Arachnid) [inline README.md]: There's nothing actually to deploy here, so this probably isn't necessary.

File: README.md (hunk 1 above)
Comment (thodges-gh) [reply README.md]: `npm` works for me. I'll get that updated.

File: README.md (hunk 2 above)
Comment (thodges-gh) [reply README.md]: Good point. This doesn't make any sense then.

**Review by Arachnid — APPROVED** [review]

**Pull request merged**